cypress/integration/models/keycloak/users/user.ts

```diff
@@ -154,10 +157,15 @@ export class User {
 
     login(): void {
         login(this.username, this.password);
+        if (this.firstLogin) {
+            this.password = Cypress.env("pass");
+            this.firstLogin = false;
+        }
     }
 
     logout() {
-        clickByText(button, this.username);
-        clickByText("a", "Logout");
+        // clickByText(button, this.username);
+        // clickByText("a", "Logout");
```

Comment:
If the logout function is enough maybe we can remove the commented code.

Reply:
You are right, I changed `logout` function and now it works in the same manner as `login`, no need to duplicate same code in several places, but I forgot to clean up commented code. Done now.